ecr_api.py

```diff
@@ -163,7 +163,7 @@ def __init__ ( self , retries=60) :
             try:
                 self.db=MySQLdb.connect(host=mysql_host,user=mysql_user,
                   passwd=mysql_password,db=mysql_db)
-            except Exception as e:
+            except Exception as e: # pragma: no cover
```

Comment:
It would be nice if this pragma was removed and instead we tested our DB database access error condition by doing a `mock` on the `MySQLdb.connect` call.